pkg/scheduler/api/job_info.go

```diff
@@ -589,3 +589,16 @@ func (ji *JobInfo) Ready() bool {
 
 	return occupied >= ji.MinAvailable
 }
+
+// IsPending returns whether job is in pending status
+func (ji *JobInfo) IsPending() bool {
```

Comment:
Maybe it can be merged with the `if` below.

Reply:
merged it